src/utils/explorer-api.js

```diff
@@ -22,7 +22,8 @@ export async function getTransactions(accountId = '') {
                     transactions.block_timestamp, 
                     actions.action_type as kind, 
                     actions.action_args as args,
-                    actions.action_index || ':' || transactions.hash as hash_with_index
+                    actions.action_index || ':' || transactions.hash as hash_with_index,
+                    :accountId as query_account_id
```

Comment:
why not just add this as a field to returned object? No need to have it in every row.

Reply:
also given that this is info only useful for reducer, it makes sense to add it in action implementation vs here.

Reply:
> why not just add this as a field to returned object?

You mean in `map` below? We can, the result will be exactly the same - there will be accountId prop for every tx.

> also given that this is info only useful for reducer, it makes sense to add it in action implementation vs here.

We can put it in `reducer` but we would have to check if accountId is  signer_id or receiver_id, which is already checked in SQL query.

Also, my intention was to make it more generic, `getTransactions` has optional `accountId` - adding it here allows us to use this function for example on Profile page, for another user than already logged in.

Reply:
>  We can, the result will be exactly the same - there will be accountId prop for every tx.

no, I mean just return an object with `{ transactions: [...], accountId: ... }` (i.e. I don't see a need to repeat this for every row). 

the result won't be the same BTW, we won't be transferring unnecessary data over wire (first from DB to explorer, then from explorer to wallet).

> getTransactions has optional accountId

Does it need to be optional vs always required? Seems like `explorer-api` doesn't have to know about concept of current account.

Reply:
> no, I mean just return an object with { transactions: [...], accountId: ... } (i.e. I don't see a need to repeat this for every row).

ok, np

Reply:
> Does it need to be optional vs always required?

good point - we can change that, in this case we would only have to add accountId for `getTransactions` action dispatched in `DashboardDetail.js`